CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2.7.1 / 2016-02-09
+
+* [CHANGE] Bump Reek to 3.10.1 (by Tomasz Pajor)
+* [BUGFIX] Analyse only the files whose paths are specified in the CLI (by Nuno Silva)
```

Comment:
Historically I can see in the changelog that we never mention the core team. 
I don't see any reason to do so but i see a gazillion reason to change this.
Let's get into the habit of mentioning ourself in the changelog for the future ;)